src/library_fs.js

```diff
@@ -795,8 +795,6 @@ mergeInto(LibraryManager.library, {
       var node = FS.lookupNode(parent, name);
       var err = FS.mayDelete(parent, name, false);
       if (err) {
-        // POSIX says unlink should set EPERM, not EISDIR
```

Comment:
please leave a comment here, that if we were POSIX compliant we would do [..], but we instead do the same thing that linux does.